### PR TITLE
Elmariuso-flags-1

### PR DESCRIPTION
**Pull request by ElMariuso**
## Changelog:
- Added flags for several orcish's titles:
  - Shattered Hand (not used)
  - Laughing Skull (not used)
  - Warsong (exists in the 603 and 605 bookmarks)
  - Black Tooth Grin
- Added flags for several religious' titles:
  - Atal'ai Tribe
  - Church of the Holy Light
  - Scarlet Monastery
- Added coat for a dynasty:
  - Shadowmoon
- Added some scripted colors:
  - Brown warsong
  - Dark purple Shadowmoon 

# Where to find the changes  ?

Warcraft-Guardians-of-Azeroth-2\gfx\coat_of_arms\colored_emblems\
Warcraft-Guardians-of-Azeroth-2\common\coat_of_arms\coat_of_arms\00_wc_landed_titles.txt
Warcraft-Guardians-of-Azeroth-2\common\coat_of_arms\coat_of_arms\wc_dynasties.txt

# How to test:
Launch the mod and search for the titles and dynasties on the title and character finder.

### CODE REVIEW

**Comment by zumbak04** [comment]
@ElMariuso assign yourself and labels.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
> Shattered Hand (not used)
> Laughing Skull (not used)
> Black Tooth Grin (not used)
> Bonechewer (not used)
> Thunderlord (not used)

@ElMariuso how to test it if it not used? And why you want to be merged if it not used?

**Comment by zumbak04** [comment]
> You can test them by adding the corresponding title to yourself by the console (except Black Tooth Grin, but I want to add the title later)

Have you tried? I haven't seen such titles.

**Review by zumbak04 — COMMENTED** [review]
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/433#issuecomment-859924374

**Comment by ElMariuso** [comment]
> 
> 
> > You can test them by adding the corresponding title to yourself by the console (except Black Tooth Grin, but I want to add the title later)
> 
> Have you tried? I haven't seen such titles.

My bad, I forgot to put it on titles.
<details>
<summary>Click to expand</summary>

image
</details>
<details>
<summary>Click to expand</summary>

image
</details>

Also, my bad, I was wrong about Bonechewer and Thunderlord, their title don't exist.
But since cultures exist, I imagine that these titles may exist one day or another, in some form.
In the meantime here are the tests done on other titles.

<details>
<summary>Click to expand</summary>

image

</details>
<details>
<summary>Click to expand</summary>

image

</details>
<details>
<summary>Click to expand</summary>

image

</details>

I also fixed problem with Scarlet Monastery flag

File: common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt [hunk 1]
```diff
@@ -279,4 +279,31 @@ d_burning_blade = {
 		color1 = "red"
```
Comment (zumbak04) [inline common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt:279]: > Shadowmoon

But this title doesn't exist.

P.S. I see it's used by Ner'zhul's dynasty, but you should mention it in the changelog.

Comment (zumbak04) [inline common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt:279]: The CoA of the Holy Light is weird.
It's pixilated and has weird color (the standard blue color is 0, 0, 128 RGB)
<details>
<summary>Click to expand</summary>

image

</details>

File: common/coat_of_arms/coat_of_arms/wc_religious_titles.txt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+### Holy Light
+d_church_of_the_holy_light = {
+	pattern = "pattern_solid.dds"
+	color1 = "white"
+	colored_emblem = {
+		texture = "ce_holy_light.dds"
+		color1 = "yellow"
+		color2 = "yellow"
+		instance = { position = { 0.5 0.4 } scale = { 0.9 0.9 }  }
```
Comment (zumbak04) [inline common/coat_of_arms/coat_of_arms/wc_religious_titles.txt:11]: Remember we talked about better not to use `instance` and correct the image instead.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt (hunk 1 above)
Comment (ElMariuso) [reply common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt:279]: edited

File: common/coat_of_arms/coat_of_arms/wc_dynasties.txt [hunk 3]
```diff
@@ -58,6 +58,15 @@ house_dragonmaw = 2401
 2250 = d_burning_blade
 house_burningblade = d_burning_blade
```
Comment (zumbak04) [inline common/coat_of_arms/coat_of_arms/wc_dynasties.txt:59]: Don't you want to make the Hellscream dynasty use the Warsong symbol?

File: common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt (hunk 1 above)
Comment (ElMariuso) [reply common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt]: New one in cabd096e0

File: common/coat_of_arms/coat_of_arms/wc_religious_titles.txt (hunk 2 above)
Comment (ElMariuso) [reply common/coat_of_arms/coat_of_arms/wc_religious_titles.txt]: fixed in a359075d0

File: common/coat_of_arms/coat_of_arms/wc_dynasties.txt (hunk 3 above)
Comment (ElMariuso) [reply common/coat_of_arms/coat_of_arms/wc_dynasties.txt:59]: I prefer to use this kind of flags for dynasties that have the same name as the clan.
As for "of Frostwolf" which has the same flag as the clan. The same goes for the "of Blackrock" dynasty which was replaced by an original flag for Blackhand.

File: common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt [hunk 4]
```diff
@@ -273,6 +273,42 @@ d_burning_blade = {
 	}
```
Comment (zumbak04) [inline common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt:273]: Your CoAs should be a bit higher, there's a mask in the guidelines. The red dot represents the center of the image.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/2D-Artist-Guidelines
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt (hunk 4 above)
Comment (zumbak04) [reply common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt:302]: Resolved by #498.

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by elcascavel — APPROVED** [review]
LGTM! I think they look great :)

**Pull request merged**